frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -518,6 +518,10 @@ const Chart = (props: ChartProp) => {
                 plconf = {};
             }
         }
+        else {
+            plconf = {};
+        }
+        plconf.displaylogo = plconf.displaylogo || false;
```

Comment:
`plconf.displaylogo = !!plconf.displaylogo` has the advantage of forcing the value to a boolean
and `else` should be on the same line as the `if` }